python/ray/tune/examples/pb2_ppo_example.py

```diff
@@ -80,7 +80,7 @@ def explore(config):
         hyperparam_bounds={
             "lambda": [0.9, 1.0],
             "clip_param": [0.1, 0.5],
-            "lr": [1e-3, 1e-5],
+            "lr": [1e-5, 1e-3],
```

Comment:
Good catch, did this work before at all?

Reply:
Nope, I don't think this is run for some reason :cry: